Packages/Status/Sources/Status/Editor/StatusEditorViewModel.swift

```diff
@@ -84,19 +84,30 @@ import SwiftUI
   var spoilerText: String = ""
 
   var isPosting: Bool = false
-  var selectedMedias: [PhotosPickerItem] = [] {
+  var mediaPickers: [PhotosPickerItem] = [] {
     didSet {
-      if selectedMedias.count > 4 {
-        selectedMedias = selectedMedias.prefix(4).map { $0 }
+      if mediaPickers.count > 4 {
+        mediaPickers = mediaPickers.prefix(4).map { $0 }
+      }
+
+      let removedIDs = oldValue
```

Comment:
Nice yeah. The fact that it was staying selected in the system picker was really bugging me off!